scripts/get_unregistered_vmware_files.py

```diff
@@ -34,55 +34,53 @@ def list_orphaned_files_per_host(host_name, host_datastore_urls, provider_key, v
             'password': credentials['host_default']['password'],
             'hostname': hostname[0]
         }
-        for datastore_url in host_datastore_urls:
-            datastore_path = re.findall(r'([^ds:`/*].*)', str(datastore_url))
-            ssh_client = SSHClient(**connect_kwargs)
-            command = 'ls ~/{}'.format(datastore_path[0])
-            exit_status, output = ssh_client.run_command(command)
-            ssh_client.close()
-            files_in_datastore = output.splitlines() if exit_status == 0 else []
-            for fil in files_in_datastore:
-                if fil not in vm_registered_files:
-                    file_type = 'UNKNOWN'
-                    number_of_files = 0
-                    command = 'test -d ~/{}/{}; echo $?'.format(datastore_path[0], fil)
-                    exit_status, output = ssh_client.run_command(command)
-                    ssh_client.close()
-                    file_extension = re.findall(r'.*\.(\w*)', fil)
-                    if file_extension:
-                        file_type = file_extension[0]
-                        number_of_files = 1
-                    if int(output.strip()) == 0:
-                        command = 'ls ~/{}/{} | wc -l'.format(datastore_path[0], fil)
+        with SSHClient(**connect_kwargs) as ssh_client:
+            for datastore_url in host_datastore_urls:
+                datastore_path = re.findall(r'([^ds:`/*].*)', str(datastore_url))
+
+                command = 'ls ~/{}'.format(datastore_path[0])
```

Comment:
We might have a problem with this one on non-root sudo user access as we use pseudo-tty in that case. But I guess we don't care that much about it now.